indico/modules/events/papers/blueprint.py

```diff
@@ -41,6 +41,8 @@
                  management.RHAssignmentListCustomize, methods=('GET', 'POST'))
 _bp.add_url_rule('/manage/papers/assignment-list/judge', 'manage_judge_papers',
                  management.RHBulkPaperJudgment, methods=('POST',))
+_bp.add_url_rule('/manage/papers/assignment-list/download', 'manage_download_papers', management.RHDownloadPapers,
```

Comment:
We could use a similar method to download a zip with the files of a particular paper revision.